### DISCUSSION TITLE
[blocks]: Activity Feed

### DISCUSSION THREAD

**Discussion by adamperlis** [shadcn-ui/ui#3953] [Blocks Request]
### Description

Can you please pretty please create something like this? 
<img width="567" alt="Screenshot 2024-06-09 at 8 50 59 AM" src="https://github.com/shadcn-ui/ui/assets/4257325/d0300422-013e-4206-b1dd-f61acaa83e8e">

### Example

https://tailwindui.com/components/application-ui/lists/feeds

**Comment by Steveb599** [comment]
Hey any news on that? Have you iimplemented by yourself maybe?

**Comment by adamperlis** [comment]
@Steveb599 i would honestly just instruct a coding assistant to help and it probably would do a good job with it. Tell it you want it to be built using shadcdn.

**Reply by Steveb599** [reply]
Will try. Did you do that already?

**Comment by adamperlis** [comment]
Haven’t yet but actually gonna do a tutorial on how to do something like it in the coming weeks. Follow along here: https://youtu.be/hAndR-38D7A?si=qsjLWJHOclhNpvID
________________________________

**Reply by Steveb599** [reply]
Actually you were right v0 and bolt.new made that up in two mins

**Comment by adamperlis** [comment]
Haha yup! Good choice for that. v0 has it basically built in with a good really good vision model and prompting and trained on Shadcdn
________________________________
